input/ignoreWarnings.txt

```diff
@@ -7,4 +7,31 @@ An HTML fragment from the set [dependency-table.xhtml, dependency-table-short.xh
 The HTML fragment 'globals-table.xhtml' is not included anywhere in the produced implementation guide
 
 # CapabilityStatement warnings 
-This element does not match any known slice defined in the profile http://hl7.org/fhir/StructureDefinition/capabilitystatement-search-parameter-combination|4.0.1
\ No newline at end of file
+This element does not match any known slice defined in the profile http://hl7.org/fhir/StructureDefinition/capabilitystatement-search-parameter-combination|4.0.1
```

Comment:
I'm not sure what the point of adding in the errors to this file? Errors are not ignored.

Reply:
Errors are not ignored. Build errors related to an artefact (e.g. ActorDefitinion errors) get annotated. Broken links and other errors are still shown including the reason included in the ignoreWarnings but you do need to scroll down to see them. I still see the value in this, and it is what other IGs do, but I do think this was premature and, if we are to do this for other messages, it should be done after the TDG acceptance of the exceptions.

Reply:
No requirement to wait for TDG approval of an exception to simply annotate the errors.  If the change to this file does that then good. 

I've seen the annotation on US Core, but did not see that equivalent annotation on the AU Core branch and thus could detect no difference to the QA report besides it declaring "22 suppressed errors".